Libraries/LibIDL/ExposedTo.h

```diff
@@ -19,7 +19,10 @@ enum class ExposedTo {
     Window = 0x10,
     ShadowRealm = 0x20,
     Worklet = 0x40,
-    AllWorkers = DedicatedWorker | SharedWorker | ServiceWorker | AudioWorklet, // FIXME: Is "AudioWorklet" a Worker? We'll assume it is for now (here, and line below)
+    PaintWorklet = 0x80,
+    LayoutWorklet = 0x100,
+    // FIXME: Are the FooWorklets Workers? We'll assume they are for now.
+    AllWorkers = DedicatedWorker | SharedWorker | ServiceWorker | AudioWorklet | PaintWorklet | LayoutWorklet,
```

Comment:
I believe `Exposed=Worklet` is a thing that exists, which might give rise to `AllWorklets` instead?

Reply:
Do things break if you leave `AllWorkers` at its original definition? 🤔